include/btllib/phred_nthash.hpp

```diff
@@ -0,0 +1,127 @@
+#ifndef BTLLIB_PHRED_NTHASH_HPP
+#define BTLLIB_PHRED_NTHASH_HPP
+
+#include "btllib/nthash.hpp"
+#include "btllib/util.hpp"
+
+#include <string>
+#include <vector>
+
+namespace btllib {
+/**
+ * NtHash with Phred score filtering.
+ */
+class PhredNtHash : private NtHash
+{
+public:
+  /**
+   * Constructor for PhredNtHash.
+   * @param seq Sequence to hash.
+   * @param seq_len Length of `seq`.
+   * @param hash_num Number of hashes to compute.
+   * @param k Length of k-mer.
+   * @param phred_min Minimum Phred score for a base to be included in the hash.

```

Comment:
I think the explanation for `phred_min` needs clarification. I thought that if the Phred score for a base is less than this threshold, the k-mer will be ignored, not that specific base (unlike spaced seeds where bases are ignored).